src/sites/dMCDA.jl

```diff
@@ -3,6 +3,10 @@
 using StatsBase
 using Distances
 using Combinatorics
+using JMcDM
+using InteractiveUtils: subtypes
+using ADRIA: order_ranking, adria_vikor, adria_topsis
```

Comment:
Quick question - was the `adria_` prefix necessary?

Looks a bit tacky in the sense that none of the other methods have the prefix.
At first glance it doesn't seem like `vikor` or `topsis` would have resulted in a clash with JMcDM functions, so wondering if we can save the extra keypresses.

Reply:
I originally didn't have the prefix but JMcDM seems to have wrapping functions for the vikor and topsis methods so instead of mcda(A,weights, method()) you can do vikor(A,weights) etc... so I had to have something different

Reply:
Okay. I still think it looks a bit clunky but leave it as is. It'll be removed in the future if you contribute changes to JMcDM anyway.